docs/examples/gallery/sbc.md

```diff
@@ -105,3 +105,47 @@ plot_ecdf_pit(sbc.simulations)
 ```
 
 :::::
+
+:::::{tab-item} Numpyro
+:sync: numpyro
+
+We define a Numpyro Model, we use the centered eight schools model.
+
+```{jupyter-execute}
+import numpyro
+import numpyro.distributions as dist
+from jax import random
+from numpyro.infer import NUTS
+y = np.array([28.0, 8.0, -3.0, 7.0, -1.0, 1.0, 18.0, 12.0])
+sigma = np.array([15.0, 10.0, 16.0, 11.0, 9.0, 11.0, 10.0, 18.0])
+def eight_schools_cauchy_prior(J, sigma, y=None):
+    mu = numpyro.sample("mu", dist.Normal(0, 5))
+    tau = numpyro.sample("tau", dist.HalfCauchy(5))
+    with numpyro.plate("J", J):
+        theta = numpyro.sample("theta", dist.Normal(mu, tau))
+    numpyro.sample("y", dist.Normal(theta, sigma), obs=y)
+# We use the NUTS sampler
+nuts_kernel = NUTS(eight_schools_cauchy_prior)
+```
+
+Pass the model to the `SBC` class, set the number of simulations to 100, and run the simulations. For numpyro model, 
+we pass in the ``data_dir`` parameter.
+
+```{jupyter-execute}
+sbc = simuk.SBC(nuts_kernel,
+    sample_kwargs={"num_warmup": 1000, "num_samples": 1000},
```

Comment:
```suggestion
    sample_kwargs={"num_warmup": 50, "num_samples": 25},
```
Here and in the other examples we are setting very low numbers for tunning, draw/samples and simulations. We do this to avoid costly computations. We should make that clear or alternatively use more realistic values and have results precomputed.

Reply:
Sounds good, done :)